app/views/orders/new.html.erb

```diff
@@ -3,6 +3,48 @@
   <%= form_for(@order) do |f| %>
```

Comment:
@dkniffin In my next PR (coming shortly) I refactor this form into three partials. That seems to help a lot - thoughts on needing to move to simple form?